### DISCUSSION TITLE
Rename App Title/Rebranding

### DISCUSSION THREAD

**Discussion by shomilsaini** [mealie-recipes/mealie#3681] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I would like to see the functionality of renaming the app from Mealie to any other name on my server. For instance, I want to name it as recipe.xyz.com for my domain and share it with family.

### Please provide a concise description of the feature that would resolve your issue.

Any environment variable or config for docker

### Please consider and list out some caveats or tradeoffs made in your design decision

None

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by zazabozaza** [comment]
> ### First Check
> 
>     * [x]  I used the GitHub search to find a similar requests and didn't find it.
> 
>     * [x]  Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered
> 
> 
> ### Please provide a concise description of the problem that would be addressed by this feature.
> 
> I would like to see the functionality of renaming the app from Mealie to any other name on my server. For instance, I want to name it as recipe.xyz.com for my domain and share it with family.
> ### Please provide a concise description of the feature that would resolve your issue.
> 
> Any environment variable or config for docker
> ### Please consider and list out some caveats or tradeoffs made in your design decision
> 
> None
> ### Additional Information
> 
>     * [ ]  If this is accepted I'm willing to submit a PR to provide this feature
> 
>     * [ ]  If this is accepted I'm willing to help maintain this feature
> 
>     * [ ]  I'm willing to sponsor/pay a developer to do this work

Hey mate, if you could provide brief instrunctions of how this could be done i would really appreciate it!
